inst/shiny/ui.R

```diff
@@ -51,7 +51,7 @@ fillPage(
                                     column(3, plotOutput("cdr3.frequency")),
                                     column(5, tableOutput("top.clonotypes"))
                                 )),
-                        column(4,
+                        column(3,
```

Comment:
Why did this change here? This has nothing to do with the merging/subsetting of objects